extensions/amp-carousel/amp-carousel.md

```diff
@@ -239,6 +236,27 @@ The visibility of arrows can also be controlled via styling, and a media query c
 }
 ```
 
+- You can position the carousel buttons using align-self and/or relative positioning. Note that the carousel arrows are automatically flipped in RTL, so you should not change their flex order.
+
+```css
+.amp-carousel-button-prev {
+  position: relative;
+  bottom: 20px;
+  align-self: flex-end;
+}
+```
+
+## Differences from `<amp-carousel>` 0.1
+
+- Autoplay is now allowed on `type="carousel"`
+- Loop is now allowed on `type="carousel"`
+
+### Migration Notes
+
+- You will need to change the required script from `amp-carousel-0.1` to `amp-carousel-0.2`.
+- The next and previous arrows are now positioned using a flex container rather than absolute positioning. If you previously had custom positioning for arrows, you will need to update your CSS.
+- The internal DOM structure of `<amp-carousel>` 0.2 is slightly different than 0.1. If you rely on CSS selectors targetting internal elements (e.g `amp-carousel > div`), you may find some of your CSS no longer works.
```

Comment:
Can we change this wording to be a bit more actionable? Something like "Ensure CSS is still valid. The internal DOM structure of `<amp-carousel>` 0.2 differs from 0.1, effecting selectors targeting internal elements, such as `amp-carousel > div`."